recipe/meta.yaml

```diff
@@ -8,8 +8,8 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.{{ bundle }}
-  sha256: 27ef00d1938e8186153867fb6f10ea2e5631db95e30677ecd9a8a556562e51b7
+  url: https://github.com/xnd-project/rbc/archive/refs/tags/v{version}.tar.gz
```

Comment:
Why the change from PyPi?

Reply:
I couldn't download the source to check the sha256

Reply:
Works on my side:

```
wget https://pypi.org/packages/source/r/rbc-project/rbc-project-0.9.1.tar.gz
```